### PR TITLE
`Exiled Ape`

### PR DESCRIPTION
**Pull request by jcstein**

### CODE REVIEW

File: Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Let's do this.
 
 ### 🎩 Going through the `mintToken` function.
 
-In your `CandyMachine` component you'll see a function named `mintToken`. This is part of Metaplex's front-end library *(huge **shoutout to the Exiled [Apes Candy Machine Mint repo](https://github.com/exiled-apes/candy-machine-mint)** for giving a good baseline for this code)*.
+In your `CandyMachine` component you'll see a function named `mintToken`. This is part of Metaplex's front-end library *(huge **shoutout to the [Exiled Apes Candy Machine Mint repo](https://github.com/exiled-apes/candy-machine-mint)** for giving a good baseline for this code)*.
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md:9]: Can you add `huge` in the bolded content please :)?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Small change I left for ya :)

File: Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md (hunk 1 above)
Comment (jcstein) [reply Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md:9]: absolutely! my apologies for the half-pull request 😅

**Review by jcstein — COMMENTED** [review]
✅

**Comment by adilanchian** [comment]
POG. Thank you for this :)

**Pull request merged**